.gitignore

```diff
@@ -2,3 +2,5 @@
 pydocstringformatter.egg-info
 dist/
 docs/_build
+venv/
+.idea/
```

Comment:
I'd prefer if you could add these to your own (global) `gitignore`. We don't really add any directories that are specific to any IDE or local environment as those should ideally be ignored by the user.

Reply:
(You can define a global gitignore so you don't have to add this to all your gitignores) https://stackoverflow.com/a/7335487/2519059